modules.json

```diff
@@ -367,4 +367,19 @@
             "ucuser": "Jimbo Wales"
     	}
     }
-]
\ No newline at end of file
+
+    {
+
+        "filename": "get_alllinks_red.py",
+        "docstring": "List all the links and check if the link is a red link or not.",
```

Comment:
Description could be something like: "Demo of `Links` module: Extract missing links from all links on a give page...". Feel free to rephrase but we want it to be consistent with other doc strings in `modules.json`.